net/instaweb/rewriter/css_inline_filter.cc

```diff
@@ -111,11 +112,22 @@ void CssInlineFilter::StartDocumentImpl() {
 
 CssInlineFilter::~CssInlineFilter() {}
 
+void CssInlineFilter::StartElementImpl(HtmlElement* element) {
+  if (element->keyword() == HtmlName::kBody) {
+    in_body_ = true;
+  }
+}
+
 void CssInlineFilter::EndElementImpl(HtmlElement* element) {
   // Don't inline if the CSS element is under <noscript>.
   if (noscript_element() != NULL) {
     return;
   }
+
+  if (element->keyword() == HtmlName::kBody) {
+    in_body_ = false;
```

Comment:
Do you see any direct benefit in setting `in_body_` to `false` again here?
I can see that doing this ensures that the value keeps reflecting the name (`in_body_`), but if that is all, maybe we should rename `in_body_` to `inlining_not_pedantically_spec_` and remove this code?